cli/generators/crud/crudGenerator.js

```diff
@@ -0,0 +1,23 @@
+var Generator = require('yeoman-generator');
+var ejs = require('ejs');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  appendTpl(templatePath, destinationPath, data) {
+    var fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
```

Comment:
No probs! I'm not so sure about yeoman fs, but I know for a fact that Node's FS should have everything we need to write and read from files -- it is a pretty basic functionality so it strikes me as surprising that there are no good libraries supporting it.